### DISCUSSION TITLE
Improvements to the SQL Editor

### DISCUSSION THREAD

**Discussion by SvKol** [supabase/supabase#27072] [Feature Requests]
There are basically 2 things, which I find 'annoying' when working with the SQL Editor:
1) You select the scripts with the mouse "on release". Which sometimes makes me incidentally open another script, while actually trying to select a bunch of code rows.
2) It'd be really nice if the scripts are sorted in "last executed, last created" order instead of alphabetical order. I sometimes create scripts, execute them, then go to another script to do something else. And I very often forget to check what name was given to the 'new' script. So if I then want to go back to it, to make adjustments, I can spend a good portion of time looking for it.

**Comment by Hallidayo** [comment]
Hi @SvKol - Have you got a video of the first point?
